Header header
sensor_msgs/Image image
std_msgs/String boxpart
int64 xmax
int64 ymax
int64 xmin
int64 ymin